user/api.go

```diff
@@ -608,7 +621,7 @@ func (a *Api) GetUserInfo(res http.ResponseWriter, req *http.Request, vars map[s
 // @Accept  json
 // @Produce  json
 // @Param userid path int true "user id for server request, from token for personal request" optional
-// @Param password body string true "password"
+// @Param password body string true "password for personal request"
```

Comment:
This parameter isn't required anymore 
```suggestion
// @Param password body string false "password mandatory for personal request"
```